lighthouse-extension/app/src/popup.js

```diff
@@ -51,7 +51,8 @@ document.addEventListener('DOMContentLoaded', _ => {
 
     let qsBody = '**Lighthouse Version**: ' + getLightHouseVersion() + '\n';
     qsBody += '**Chrome Version**: ' + getChromeVersion() + '\n';
-    qsBody += '**Stack Trace**:\n' + err.stack;
+    qsBody += '**Error Message**: ' + err.message + '\n';
+    qsBody += '**Stack Trace**:\n ```' + err.stack + '```';
```

Comment:
The URL the user was running LH against would also be helpful to add to the bug report.